_test/disabled_tests.md

```diff
@@ -1,40 +1,42 @@
 # Disabled tests
 
 - test_change_crystal_1a.m
-	- test_u_alighnment_tf_way (no ticket)
-	- test_u_alighnment (no ticket)
+        - test_u_alighnment_tf_way (no ticket) -- Why disasbled?
+        - test_u_alighnment (no ticket)
 
-- test_gen_sqw_powders.m
-	- test_combine_cyl_tot (no ticket)
+- test_gen_sqw_powders.m - Not called that? test_gen_sqw_powders/test_combine_cyl.m
+        - test_combine_cyl_tot (no ticket) -- Not disabled?
 
 - test_combine_pow.m
-	- test_combine_pow_tot (no ticket)
+        - test_combine_pow_tot (no ticket) -- Not disabled?
```

Comment:
These two certainly look like they're not disabled any more. Probably best to take them off of this list.

Reply:
My plan was to render the list obsolete because we can get the skipped tests by looking at the verbose log of skipped tests.

Reply:
Or simply searching for "skipTest" as it's a standard name now.